.github/config/wordlist.txt

```diff
@@ -0,0 +1,290 @@
+accessmethod
+accessmethoddescriptor
+accessmethods
+actiondescriptor
+additionalresource
+addversion
+adr
+aggregative
+aml
+anchore
+api
+archlinux
+artifactid
+aur
+auth
+autoconfigure
+aws
+backend
+backends
+basename
+bindata
+blobaccess
+blobhandler
+bool
+boolean
+buildx
+cas
+cli
+cliconfig
+clis
+closable
+closeable
+cmds
+cncf
+codeql
+commontransportarchive
+componentaccess
+componentaccessimpl
+componentarchive
+componentreferences
+componentversionaccess
+componentversionaccessimpl
+componentversions
+compvers
+config
+configfile
+configs
+consumerid
+containerd
+crd
+creationtime
+credentialrequest
+credentialsspec
+crossplatform
+crypto
+ctf
+ctfs
+ctx
+cvs
+deepcopy
+defs
+dependabot
+deployscript
+deserializable
+deserialization
+deserialize
+deserialized
+dest
+digesters
+directorytree
+dirtree
+dns
+dockerconfig
+downloadconstraint
+downloader
+downloaderdescriptor
+downloaders
+downloadhandlers
+dry-run
+dup
+dups
+dynaml
+echoserver
+entrypoint
+executables
+executoroptions
+executorspecification
+extensibility
+filepath
+filesystem
+filesytem
+finalizer
+finalizers
+foldername
+forwardspec
+gapped
+gav
+getinexpensivecontentversionidentity
+getresourcesbyname
+getter
+ghcr
+github
+gitignore
+gitops
+globalaccess
+globals
+gocritic
+gofmt
+golang
+golangci
+gomega
+goreleaser
+gosec
+goutils
+groupid
+gzip
+handleoutput
+hashers
+hashicorp
+hashicorpvault
+helmchart
+helmchartrepository
+helmdemo
+helmexecutor
+helminstaller
+helmrepository
+hoc
+homebrew
+hostname
+hostpath
+html
+http
+https
+iaas
+imagemapping
+inlined
+installable
+instantiation
+io
+ioutils
+iterable
+json
+jsonschema
+keyless
+keypair
+keyring
+kubebuilder
+kubeconfig
+kubernetes
+labelmergespecification
+lifecycle
+linter
+linters
+lister
+listers
+localizations
+logconfig
+logrus
+lookups
+macos
+makefile
+makefiles
+mandelsoft
+mapocirepoattr
+matcher
+matchers
+mavenartifact
+mavenpackage
+mavenrepository
+mediatype
+mimetype
+mimetypes
+multiarch
+mutex
+mvn
+mvnrepository
+namespace
+namespaces
+newcomponentaccess
+newcomponentversionaccess
+newrepository
+newrepositoryspec
+npm
+npmjs
+npmrc
+npmregistry
+oauth
+oci
+ociartifact
+ocireg
+ociregistry
+ocm
+ocmbot
+ocmcli
+ocmconfig
+oncefunc
+oo
+oras
+os
+outputspecification
+packagespec
+parameterization
+pendingdeprecationwarning
+pflag
+podinfo
+podman
+pre
+preconfigure
+preconfigured
+predefine
+programmatically
+prs
```

Comment:
The occurrences of `prs` are `PRs`. Maybe we should just write `pull-requests` instead of adding `prs` to the wordlist to prevent any real typo of `prs`.

Reply:
Although there' the chance of misspelling PRs as Pre or pre or any other permutation, I don't want to always write Pull-request instead of PRs and am willing to live with potential "errors" here :-)